modulefiles/tasks/hera/pre_post_stat.local.lua

```diff
@@ -1 +1 @@
-load(pathJoin("nco", os.getenv("nco_ver") or "4.9.3"))
+load(pathJoin("nco", os.getenv("nco_ver") or "5.1.6"))
```

Comment:
Should this be 5.0.6?

```suggestion
load(pathJoin("nco", os.getenv("nco_ver") or "5.0.6"))
```